Dockerfile

```diff
@@ -1,12 +1,14 @@
 FROM ubuntu:18.04
 
+COPY . style-analyzer
+
 RUN apt-get update && \
     apt-get install -y --no-install-suggests --no-install-recommends ca-certificates locales \
-      libxml2 libxml2-dev libsnappy1v5 libsnappy-dev gcc g++ wget \
+      libgomp1 libxml2 libxml2-dev libsnappy1v5 libsnappy-dev gcc g++ wget \
       python3 python3-dev python3-distutils && \
     wget -O - https://bootstrap.pypa.io/get-pip.py | python3 && \
-    pip3 install --no-cache-dir sourced-ml && \
-    pip3 uninstall -y pyspark && \
+    cd style-analyzer && \
+    pip3 install -r requirements.txt && \
```

Comment:
We fixed version of `bblfsh` package for style-analyzer, so we need to have `g++` when we install it, so I move it here.

It was the main reason for docker image failure.